README.md

```diff
@@ -68,7 +68,7 @@ use `$ ps aux` to locate the node process, terminate it, then restart it with `$
 Must include `config.js` and `auth-server/config.php` (not included in repo) to work.
 
 
-##Bypass Certificate Authentication
+## Bypass Certificate Authentication
```

Comment:
there's a merge conflict in the readme which prevents the pr from being merged, can you resolve it?